c_api/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
+
+project(faiss_c_library LANGUAGES C CXX)
+
+set(CMAKE_C_STANDARD 11)
+
+file(GLOB SOURCES "${CMAKE_SOURCE_DIR}/c_api/*.cpp")
```

Comment:
Please do not use `file(GLOB)` for sources as adding or removing a source file will not be caught by the generated build files. Instead, list the files manually in a variable (`set(FAISS_C_SRC ...)`).

Reply:
Also, please replace `${CMAKE_SOURCE_DIR}/c_api/` with `${PROJECT_SOURCE_DIR}/`.